deepchem/models/tensorgraph/tensor_graph.py

```
@@ -519,14 +524,17 @@ def _install_queue(self):
       for layer in self.features + self.labels + self.task_weights:
         layer.pre_queue = True
       return
+    inputs = self.features + self.labels + self.task_weights
+    if len(inputs) == 0:
+      return
```

Why?  Do we expect this case?  Should we raise an Exception instead?